app/controllers/mfa_confirmation_controller.rb

```diff
@@ -49,34 +32,6 @@ def password
     params.require(:user)[:password]
   end
 
-  def handle_valid_password
-    if current_user.auth_app_configurations.any?
-      redirect_to login_two_factor_authenticator_url(reauthn: true)
-    else
-      redirect_to user_two_factor_authentication_url(reauthn: true)
-    end
-    session[:password_attempts] = 0
-    user_session[:current_password_required] = false
```

Comment:
There's another reference here, should it be removed?

https://github.com/18F/identity-idp/blob/daa67d01b82180e20a4681c7f69236296910eda1/app/controllers/password_capture_controller.rb#L41

Reply:
Yep, thank you, removed in 68500c1